src/os2datascanner/projects/admin/adminapp/templates/miniscan.html

```diff
@@ -56,6 +62,21 @@ <h2>{% trans "File to scan" %}</h2>
               {% trans "Please, select another (smaller) file to scan." %}
             </div>
           {% endwith %}
+<<<<<<< HEAD
```

Comment:
Git leftover

Reply:
Same as other git merge problem. Thanks again for noticing !